build_pio.rs

```diff
@@ -1,84 +0,0 @@
-//! Install tools and build the `esp-idf` using `platformio`.
```

Comment:
Maybe this file should have been renamed (like `build_native.rs`) for history consistency.

Reply:
It is renamed to just `pio.rs` actually. The patch is somehow messed up. Will fix.